internal/distro/rhel8/images.go

```diff
@@ -466,6 +466,17 @@ func edgeSimplifiedInstallerImage(workload workload.Workload,
 	rawImg.Groups = users.GroupsFromBP(customizations.GetGroups())
 
 	rawImg.KernelOptionsAppend = []string{"modprobe.blacklist=vc4"}
+
+	var err error
```

Comment:
Same here for the generalization

Reply:
Commented in the Fedora distro. TL;DR; I'm not in favor of generalizing this.